### ISSUE TITLE
Migration to multi label long tailed recognition

### ISSUE THREAD

**Issue by ZYang2077** [KaihuaTang/Long-Tailed-Recognition.pytorch#30]
Thank you for your excellent work. I want to try to migrate the classifier (CausalNormClassifier) in this work to the task of multi label long tailel recognition. However, the following problems appeared. I believe the model has not learned any knowledge at all, perhaps the output is smoothed out in the inference stage. Because the loss change during training seems normal. Can you give me some advice or tell me the possible problems?

感谢你开源这么优秀的工作，我想尝试把这个工作中的分类器迁移到多标签长尾数据分类任务中去，但是却出现了如下的问题，感觉模型完全没有学习到任何知识，或者好像是在推理的阶段输出被平滑抵消掉了？因为在训练过程中的loss变化看起来是正常的。能否给我一些建议或者告诉我可能出现的问题？

![捕获](https://user-images.githubusercontent.com/58316756/115868254-b14b8e80-a42b-11eb-8c48-8d9761d6437d.PNG)

**Comment by ZYang2077** [comment]
Could it be the difference between softmax and sigmoid that affects the results？

**Comment by jsetty** [comment]
> Could it be the difference between softmax and sigmoid that affects the results？

Hi @AlphaPlusTT ,

Did you manage to find a solution for multi-label long tail classification?

Thanks

**Comment by jsiddamsetty** [comment]
> Thank you for your excellent work. I want to try to migrate the classifier (CausalNormClassifier) in this work to the task of multi label long tailel recognition. However, the following problems appeared. I believe the model has not learned any knowledge at all, perhaps the output is smoothed out in the inference stage. Because the loss change during training seems normal. Can you give me some advice or tell me the possible problems?
> 
> 感谢你开源这么优秀的工作，我想尝试把这个工作中的分类器迁移到多标签长尾数据分类任务中去，但是却出现了如下的问题，感觉模型完全没有学习到任何知识，或者好像是在推理的阶段输出被平滑抵消掉了？因为在训练过程中的loss变化看起来是正常的。能否给我一些建议或者告诉我可能出现的问题？
> 
> ![捕获](https://user-images.githubusercontent.com/58316756/115868254-b14b8e80-a42b-11eb-8c48-8d9761d6437d.PNG)

Hi @AlphaPlusTT ,

Did you manage to find a solution for multi-label long tail classification?

Thanks
